forc/src/utils/manifest.rs

```diff
@@ -109,8 +111,9 @@ fn try_parse_contract_id(raw: &Option<String>) -> Result<fuel_tx::ContractId, St
             })?
             .as_slice(),
     )
-    .unwrap())
+        .unwrap())
```

Comment:
Hmm. Why is this formatting different? The code should have been formatted with `cargo fmt`, no?

Reply:
Weird because I did run `cargo fmt`, and the fmt CI check is marked as passing.

Reply:
Ok the issue was that I didn't run it with the `--all` flag, thanks for pointing that out